testing/junit/src/main/java/com/linecorp/armeria/testing/junit/server/mock/RecordedRequest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit.server.mock;
+
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+
+import com.google.common.base.MoreObjects;
+
+import com.linecorp.armeria.common.AggregatedHttpRequest;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+/**
+ * A request that has been made to a {@link MockWebServerExtension}.
+ */
+public final class RecordedRequest {
+
+    private final AggregatedHttpRequest request;
+    private final ServiceRequestContext context;
+
+    public RecordedRequest(AggregatedHttpRequest request, ServiceRequestContext context) {
```

Comment:
Sorry, but how about putting context first for consistency with other methods in our API?

Reply:
Sure